packages/frontend/src/components/pages/search-page/searchPage_service.js

```diff
@@ -7,4 +7,13 @@ const getOrder = async (searchId) => {
   return response.data
 }
 
-export default getOrder
+const findOrder = async (searchData) => {
+  if (searchData.id) {
+    Object.assign(searchData, { id: +searchData.id })
+  }
+  const url = getFullApiUrl('search-orders')
+  const response = await axios.post(url, { filters: searchData })
+  return response.data
```

Comment:
И давай вернем сразу массив заказов тут
```suggestion
  return response.data.orders
```